typesense-java/src/main/java/org/typesense/api/CollectionResponse.java

```diff
@@ -0,0 +1,49 @@
+package org.typesense.api;
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.typesense.api.CollectionSchema;
+
+public class CollectionResponse extends CollectionSchema {
+
+   /**
+     * Number of documents in the collection
+     **/
+    private Long numDocuments = null;
+    /**
+     * Number of documents in the collection
+     * @return numDocuments
+     **/
+    @JsonProperty("num_documents")
+    public Long getNumDocuments() {
+        return numDocuments;
+    }
+
+    @JsonProperty("created_at")
+    public String created_at;
+
+    @JsonProperty("num_memory_shards")
+    public long num_memory_shards;
+
+
+    @Override
+    public String toString() {
```

Comment:
I can't recall if I left commons-lang in as a dependency, but might be able to use the ToStringBuilder instead.